code/game/objects/structures/disaster_counter.dm

```diff
@@ -0,0 +1,96 @@
+/**
+  * # Disaster counter.
+  *
+  * Tracks how many shifts it has been since the counter with that ID was exploded.
+  */
+/obj/structure/disaster_counter
+	name = "disaster counter"
+	desc = "This device will counts how many shifts it has been since a major disaster in this area. A safe workplace is a productive workplace."
```

Comment:
```suggestion
	desc = "This device will count how many shifts it has been since a major disaster in this area. A safe workplace is a productive workplace."
```